src/helpers/showTypeUtils.ts

```diff
@@ -37,20 +37,45 @@ const convertResultsToShowType = (
             release_date:
                 show.media_type === 'movie'
                     ? (show as MovieData).release_date
-                    : (show as TvData).first_air_date
+                    : (show as TvData).first_air_date,
         };
     });
 
     return filterShowsByType(showData, 'both');
 };
 
+const convertResultsForIntheaters = (
```

Comment:
How is this different from the function on line 21? `convertResultsToShowType`

Reply:
The function on line 21 further filters the data which we do not need for the movies in theaters. using that will return an empty array

Reply:
Ah sorry I see the confusion here. 

`convertDetailsToShowType` is what is needed here since that ads in the media type.

Reply:
Ok I will check it out in a bit and give you feedback

Reply:
`convertDetailsToShowType` returns a single ShowData, so that will also not work since we are trying to return an array of ShowData. @Thenlie . With the current implementation that I have pushed that works perfectly